apps/dashboard/src/components/billing/plan-action-button.tsx

```diff
@@ -31,7 +33,7 @@ export function PlanActionButton({
     if (isPaidSubscriptionActive()) {
       navigateToPortal();
     } else {
-      navigateToCheckout(selectedBillingInterval);
+      navigateToCheckout({ billingInterval: selectedBillingInterval, requestedServiceLevel: checkOutServiceLevel });
```

Comment:
If we rename the component props we will save a few characters.
```suggestion
      navigateToCheckout({ billingInterval, requestedServiceLevel });
```